packages/userprofile/README.md

```diff
@@ -4,6 +4,15 @@
 [![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepuserprofile.svg)](https://www.npmjs.com/package/@adobe/react-native-aepuserprofile) 
 [![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepuserprofile)](https://www.npmjs.com/package/@adobe/react-native-aepuserprofile)
 
+## Prerequisites
+
+The UserProfile extension has the following peer dependency, which must be installed prior to installing the UserProfile extension:
+- [Core](../core/README.md)
```

Comment:
Change to absolute path so can access from npm?

Reply:
Actually, I have verified that the relative path also works, check this out -> https://www.npmjs.com/package/@adobe/react-native-aepmessaging (messaging is also using the relative path in  the README)

Reply:
Good to know that!